Source/SaveStates/SaveState.cs

```diff
@@ -275,6 +281,7 @@ private IEnumerator LoadStateCoro(bool loadDuped)
             typeof(HeroController)
                 .GetMethod("FinishedEnteringScene", BindingFlags.NonPublic | BindingFlags.Instance)?
                 .Invoke(HeroController.instance, new object[] {true, false});
+            ReflectionHelper.CallMethod(GameManager.instance, "UpdateUIStateFromGameState", Array.Empty<object>());
```

Comment:
You can leave the last parameter blank if it doesn't need any parameters so like `ReflectionHelper.CallMethod(GameManager.instance, "UpdateUIStateFromGameState")`